### DISCUSSION TITLE
Suggestion: add match statistics and referee data

### DISCUSSION THREAD

**Discussion by fqueiro** [dcaribou/transfermarkt-datasets#243] [General]
It would be great to have data on match statistics and referees, as e.g. here:
https://www.transfermarkt.com/gd-chaves_fc-vizela/statistik/spielbericht/4109381

I could not find this, is it available?

**Comment by dcaribou** [comment]
Hey @fqueiro,
The game referee name is available in the games asset (`games` > `referee`).

As for the match statistics, you are correct this is not currently extracted.

**Comment by fqueiro** [comment]
Great, thanks @dcaribou. One statistic I was looking for specifically is the number of fouls, is this available anywhere?

**Reply by dcaribou** [reply]
Number of fouls isn't currently available unfortunately.

I does sound like a useful addition. I've logged in an issue for it (https://github.com/dcaribou/transfermarkt-datasets/issues/244)
